src/main/java/fr/esgi/al5_2/Tayarim/services/AdministrateurService.java

```diff
@@ -0,0 +1,171 @@
+package fr.esgi.al5_2.Tayarim.services;
+
+import fr.esgi.al5_2.Tayarim.dto.proprietaire.*;
+import fr.esgi.al5_2.Tayarim.entities.Administrateur;
+import fr.esgi.al5_2.Tayarim.entities.Proprietaire;
+import fr.esgi.al5_2.Tayarim.exceptions.*;
+import fr.esgi.al5_2.Tayarim.mappers.AdministrateurMapper;
+import fr.esgi.al5_2.Tayarim.mappers.ProprietaireMapper;
+import fr.esgi.al5_2.Tayarim.repositories.AdministrateurRepository;
+import fr.esgi.al5_2.Tayarim.repositories.ProprietaireRepository;
+import lombok.NonNull;
+import org.mindrot.jbcrypt.BCrypt;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'java.util.List' import. Should be before 'org.springframework.transaction.annotation.Transactional'.